docs/whitepaper/src/opening-consensus-group-and-leaders.tex

```diff
@@ -0,0 +1,80 @@
+        \item Then, to claim an initial stakes $\{s_i\}_{i=0}^M$ every protocol participant sends a request ${(\textbf{ver\_keys}, sid, PK_i, v_i^{vrf}, v_i^{kes}, v_i^{dsig})}$ to ${\mathcal_{F}}_{VRF}^{Init}$, which saves the key tuple ${(PK_i, v_i^{vrf}, v_i^{kes}, v_i^{dsig})}$.
+        \item Set of the verification keys tuples ${\{(PK_i, v_i^{vrf}, v_i^{kes}, v_i^{dsig})\}_{i=0}^M}$ \textbf{should be stored in the blockchain and acknowledged by all members of the consensus group}.
+        \item Finally ${\mathcal_{F}}_{VRF}^{Init}$ is evaluated to sample a random value $\eta \leftarrow \{0, 1\}^\lambda$ and constructs a genesis block ${B_0=\left(\{(PK_i, v_i^{vrf}, v_i^{kes}, v_i^{dsig}, s_i)\}_{i=0}^M, \eta\right)}$ \textbf{which should also be agreed upon all stakeholders}.
+    \end{enumerate}
+    \item \textbf{Epoch}.
+    Each epoch supposed to consistent of $K \in \mathbb{N}$ slots.
+    For every slot ${sl_k\in e_j}$:
+    \begin{enumerate}
+        \item Every online consensus group member collects en existed chains and verifying that for every chain every block, produced up to $Z$ blocks before contains correct data about slot $sl'$ leader $PK'$.
+        To verify a valid slot leader, response from the ${\mathcal_{F}}_{VRF}$ to query ${(\textbf{Verify}, sid, \eta' || sl', y', \pi', v^{vrf'})}$ should be ${(\textbf{Verified}, sid, \eta' || sl', y', \pi', 1)}$ and $y'<T'$ as well.
+        \item Then every lottery participant separately evaluates ${\mathcal_{F}}_{VRF}$ with his own inputs ${(\textbf{EvalProve}, sid, \eta || sl_k)}$.
+        Received output ${(\textbf{Evaluated}, sid, y_i, \pi_i)}$ includes generated random number $y_i$ and the associated proof $\pi_i$.
+        \item If ${y_i < T_i}$, where $T_i$ is a leader election threshold for this stakeholder, then $PK_i$ is a slot leader.
+        The ${\mathcal_{F}}_{VRF}$ is designed in such a way that not every slot has a leader, moreover, most of the slots remain empty to serve protocol synchronization.
+        \item If there are $P$ several elected leaders for this slot, they all propose a new blocks $\{B_p\}_{p=0}^P$ with included proofs of their leadership ${(v^{vrf^*} y^*, \pi^*)}$.
+        All forks will be further resolved according to the longest chain rule.
+        \item At the end of an epoch, each participant looks at the first $2/3\cdotK$ blocks of the epoch and concatenate all the $y'_k$ values together along with the $\eta$, hashing the result produces the $\eta^*$ for the upcoming epoch.
+        Each participant also updates their $T_j$ values to reflect the stake distribution at the start of the just completed epoch.
+    \end{enumerate}
+\end{enumerate}
+
+Opening consensus group lottery is similar and bases on the main principles described above.
```

Comment:
This transition to the actual topic is slightly counter-intuitive. Not clear what protocol was described above if we came to "opening consensus group lottery" only now.